lib/deps.js

```diff
@@ -54,8 +54,18 @@ const loadModule = (name) => {
   const subKeys = Object.keys(pkg.exports).map((key) => key.substring(2));
   const subNames = subKeys.filter(validSubmodules);
   for (const subName of subNames) {
-    const sub = appRequire(name + '/' + subName);
-    lib[subName] = sub;
+    try {
+      const sub = appRequire(name + '/' + subName);
+      if (lib[subName] && lib[subName] === sub[subName]) continue;
+      lib[subName] = sub;
+    } catch (e) {
+      if (e.message.startsWith("Cannot find module '")) {
```

Comment:
- We do not use double quotas `"`, use linter before commit
- Parsing error in a such way is not reliable, it's better to get text between two double quotas with `metautil.between(e.message, '"', '"')` see: https://github.com/metarhia/metautil/tree/master#strings-utilities
- `e` is a bad naming, better `err` or even `error`

Reply:
I used linter before commit.
It didn't fix the quotes.
Looks like prettier wants to use double quotes if string contains a single quote character: `'`
![image](https://github.com/metarhia/impress/assets/5096735/4a8ef3f9-c8f4-404a-a0d3-18a3d77efdcf)